script.skipit/resources/lib/skipit.py

```diff
@@ -92,8 +123,15 @@ def skip_backward():
     halve()
     set_label(skip_secs, False)
     # Add a couple of extra seconds as the video is still playing forwards
-    command = build_command(skip_secs + slippage, False)
-    xbmc.executebuiltin(command)
+    requested_skip = - skip_secs + slippage
+    actual_skip = do_skip(requested_skip);
+    if actual_skip != requested_skip:
+        skip_secs = abs(actual_skip)
+        text = ADDON.getLocalizedString(32130)
+        command = 'Notification({},{})'
```

Comment:
Why not use `xbmcgui.Dialog().notification(heading, message[, icon, time, sound])`: https://codedocs.xyz/MartijnKaijser/xbmc/group__python___dialog.html#ga27157f98dddb21b44cc6456137977aa2

Reply:
I would prefer that. Previously I couldn't fathom the api for that one. Thanks